release.sh

```diff
@@ -145,21 +153,43 @@ if [ $dodist -eq 0 ]; then
     # We haven't tagged yet, so tell the dist script what version
     # it's building
     DIST_VERSION="$tag" npm run dist
+
+    `dirname $0`/scripts/changelog_head.py > latest_changes.md
```

Comment:
except that at this point you are in the build dir, so node_modules won't be found